lib/config/options/index.ts

```diff
@@ -1188,6 +1188,14 @@ const options: RenovateOptions[] = [
     cli: false,
     env: false,
   },
+  {
+    name: 'autoReplaceRegexGlobalMatch',
+    description:
+      'Use to control whether replacement regular expressions are global matches or only the first match.',
+    type: 'boolean',
+    stage: 'branch',
```

Comment:
do we need the stage field?

Reply:
No, i think that is now redundant.  it was needed when it was under parent: packageRules.  Removed